src/facade/dragonfly_connection.h

```diff
@@ -116,11 +125,16 @@ class Connection : public util::Connection {
   std::unique_ptr<ConnectionContext> cc_;
 
   struct Request;
+  struct DispatchOperations;
```

Comment:
Unless I missed anything, DispatchOperations and DispatchCleanup could reside in cc file and do not have to be part of the connection class.

Reply:
I need them there because they require access to the private part of the enclosing class. I tried otherwise and failed to compile it. They are not really part of the interface since you cannot use them from the outside and they are in the private part of the class.